.github/workflows/nightly.yml

```diff
@@ -373,106 +375,57 @@ jobs:
           echo opcache.jit_hot_func=1 >> /etc/php.d/opcache.ini
           echo opcache.jit_hot_return=1 >> /etc/php.d/opcache.ini
           echo opcache.jit_hot_side_exit=1 >> /etc/php.d/opcache.ini
+          echo opcache.jit_blacklist_root_trace=255 >> /etc/php.d/opcache.ini
+          echo opcache.jit_blacklist_side_trace=255 >> /etc/php.d/opcache.ini
```

Comment:
It's better to keep the default values.

Reply:
Personally, I would prefer to split this PR into two.
1. Change ini setting
2. Add new apps (this should be merged only after the first passes nightly tests)

I'm not sure if switching from yml tasks description to nightly.php is good or bad, but I can't follow the changes...

Reply:
@dstogov Done! https://github.com/php/php-src/pull/12662

Switching to the new nightly.php script allows to parallelize the phpunit tests of all the various libraries (this yields considerable speed improvements especially with CI workers with a lot of CPU cores); plus using a simple PHP script instead of a platform-dependent job declaration allows easily running the nightly tests locally and/or on different CIs (like for example, it might be worth running at least the nightly.php script on the ARM CI as well to test arm JIT)